src/Pretzel.Tests/Templating/Context/SiteContextGeneratorTests.cs

```diff
@@ -791,21 +807,25 @@ public void page_with_false_date_in_title()
         [Fact]
         public void post_default_values()
         {
-            fileSystem.AddFile(@"C:\TestSite\_posts\SomeFile.md", new MockFileData(@"---
+            var filename = @"C:\TestSite\SomeFile.md";
```

Comment:
Well the file path was modified from "C:\TestSite\_posts\SomeFile.md" to "C:\TestSite\SomeFile.md" so it's normal that Posts is empty, but why change the file path?